[supervisord]
nodaemon=true

[program:data2cache_once]
command = /usr/local/bin/python -u modules/foot/data2cache.py
stdout_logfile=/var/log/qa_data_job/data2cache_once.log
stdout_logfile_maxbytes=0
stderr_logfile=/var/log/qa_data_job/data2cache_once_error.log
stderr_logfile_maxbytes=0

[program:qa_data_job]
command = /usr/local/bin/python -u modules/foot/qa_data_job.py
stdout_logfile=/var/log/qa_data_job/access.log
stdout_logfile_maxbytes=0
stderr_logfile=/var/log/qa_data_job/error.log
stderr_logfile_maxbytes=0

[program:uwsgi]
command=/usr/local/bin/uwsgi --ini /QAdemo/conf/uwsgi.ini
stdout_logfile=/var/log/uwsgi/access.log
stdout_logfile_maxbytes=0
stderr_logfile=/var/log/uwsgi/error.log
stderr_logfile_maxbytes=0

[program:nginx]
command=/usr/sbin/nginx
stdout_logfile=/var/log/nginx/access.log
stdout_logfile_maxbytes=0
stderr_logfile=/var/log/nginx/error.log
stderr_logfile_maxbytes=0
# Graceful stop, see http://nginx.org/en/docs/control.html
stopsignal=QUIT

[program:redis_local]
command=redis-server /etc/redis.conf
autostart=true
autorestart=true
user=root
stdout_logfile=/var/log/redis/stdout.log
stderr_logfile=/var/log/redis/stderr.log